<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="configTab">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="configScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="configScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>691</width>
             <height>405</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="robotTab">
       <attribute name="title">
        <string>Robot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="robotScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="robotScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>691</width>
             <height>405</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="refreeTab">
       <attribute name="title">
        <string>Refree</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="refreeScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="refreeScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>691</width>
             <height>405</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="insertTab">
       <attribute name="title">
        <string>Insert</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="configGroupBox">
          <property name="title">
           <string>config</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>STRING:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lable_10">
             <property name="text">
              <string>VALUE:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>NAME:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="configcomboBox">
             <item>
              <property name="text">
               <string>Line Edit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combo Box</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Radio Button</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Check Box</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="configId"/>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="configName"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="configString"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>TYPE:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="configValue"/>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QWidget" name="widget" native="true">
             <widget class="QPushButton" name="configRemove">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
             <widget class="QPushButton" name="configAdd">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>ADD</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="robotGroupBox">
          <property name="title">
           <string>robot</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QLineEdit" name="robotString"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="robotId"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>STRING:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="robotName"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="robotComboBox">
             <item>
              <property name="text">
               <string>Line Edit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combo Box</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Radio Button</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Check Box</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>TYPE:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>NAME:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="robotValue"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>VALUE:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QWidget" name="widget_2" native="true">
             <widget class="QPushButton" name="robotRemove">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
             <widget class="QPushButton" name="robotAdd">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>ADD</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="refreeGroupBox">
          <property name="title">
           <string>refree</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>NAME:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>STRING:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>VALUE:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="refreeName"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="refreString"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>TYPE:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="refreeValue"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="refreeId"/>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="refreeComboBox">
             <item>
              <property name="text">
               <string>Line Edit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combo Box</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Raido Butto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Check Box</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QWidget" name="widget_3" native="true">
             <widget class="QPushButton" name="refreeRemove">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
             <widget class="QPushButton" name="refreeAdd">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>0</y>
                <width>89</width>
                <height>25</height>
               </rect>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Information</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="loadLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="saveLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOMID">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actiongithub"/>
    <addaction name="separator"/>
    <addaction name="actionwebsite"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actionset_load_database"/>
    <addaction name="separator"/>
    <addaction name="actionset_save_load_database"/>
   </widget>
   <addaction name="menuOMID"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiongithub">
   <property name="text">
    <string>github</string>
   </property>
  </action>
  <action name="actionwebsite">
   <property name="text">
    <string>website</string>
   </property>
  </action>
  <action name="actionset_load_database">
   <property name="text">
    <string>set load database</string>
   </property>
  </action>
  <action name="actionset_save_load_database">
   <property name="text">
    <string>set save load database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
